File: 프로그래머스/lv1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.js
function solution(n, m) {
    let num;
    let gcd;
    if (n < m) {
        num = m;
        gcd = n;
    } else {
        num = n;
        gcd = m;
    }
    
    while (num % gcd !== 0) {
        let remainder = num % gcd;
        num = gcd;
        gcd = remainder;
    }
    
    return [gcd, n * m / gcd];
}

File: 백준/Bronze/2609. 최대공약수와 최소공배수/최대공약수와 최소공배수.cc
#include <iostream>

using namespace std;

int main()
{
	int num1, num2;
	cin >> num1 >> num2;

	int num, gcd;
	if (num1 < num2)
	{
		num = num2;
		gcd = num1;
	}
	else
	{
		num = num1;
		gcd = num2;
	}

	while (num % gcd != 0)
	{
		int remainder = num % gcd;
		num = gcd;
		gcd = remainder;
	}

	cout << gcd << '\n' << num1 * num2 / gcd;
}